app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -147,6 +148,27 @@ class PinPasswordActivityTest {
     }
   }
 
+  @Test
+  fun testPinPassword_pinView_hasContentDescription() {
+    ActivityScenario.launch<PinPasswordActivity>(
+      PinPasswordActivity.createPinPasswordActivityIntent(
+        context = context,
+        adminPin = adminPin,
+        profileId = adminId
```

Comment:
`profileId = adminId` this doesn't sounds good. Can we have a good first issue for this?

Reply:
I think this is correct. What do you suggest instead?

Reply:
All tests passed. Resolving this.